process/receiptslog/trieRecreate.go

```diff
@@ -1 +1,24 @@
 package receiptslog
+
+import (
+	"github.com/multiversx/mx-chain-go/common"
+	"github.com/multiversx/mx-chain-go/common/holders"
+	"github.com/multiversx/mx-chain-go/storage"
+	"github.com/multiversx/mx-chain-go/trie"
+)
+
+// RecreateTrieFromDB will recreate the trie from the provided storer
```

Comment:
Why is this not inside `trieInteractor.go` ?

Reply:
moved there